po/mediawriter.pot

```diff
@@ -1251,6 +1251,42 @@ msgid ""
 "at all."
 msgstr ""
 
+#: [i3].subvariant
+msgctxt "Release|"
+msgid "i3"
+msgstr ""
+
+#: [i3].category
+msgctxt "Release|"
```

Comment:
This has to be removed/empty, otherwise it will be a duplicated string to translate. See other spins.

Reply:
Sure, I will remove that.

Reply:
I'm a bit confused here: the other spins also contain the line `msgctxt "Release|"`, so should it really be removed just for i3?

Reply:
No, this one is category and it has to remain be the same.

Reply:
Aha, it's still not removed.

Reply:
I'm sorry, I really have no clue what I should do with the `.pot` file. Would you mind using github's suggestion feature to indicated what you mean or push directly to this PR?

Reply:
Just remove:
```
msgctxt "Release|"
msgid "spins"
msgstr ""
```
and keep `#: [i3].category` comment.

Reply:
Apologies, now I get what you meant… Should be fixed now. Thanks for your patience!